DistributionPackages/Neos.NeosIo/Resources/Private/Fusion/NodeTypes/VideoEmbed.fusion

```diff
@@ -0,0 +1,14 @@
+prototype(Neos.NeosIo:VideoEmbed) < prototype(Neos.Neos:ContentComponent) {
+
+    mp4src = Neos.Fusion:ResourceUri {
+        resource = ${q(node).property('mp4').resource}
+    }
+
+    renderer = afx`
+        <video class="videoEmbed" autoplay muted loop="true" playsinline="true">
+            <source type="video/mp4" src={props.mp4src}/>
+        </video>
+    `
+
+    renderer.@process.notSet = ${props.mp4src == '' ? '<p>Please select a video in the inspector</p>' : value}
+}
```

Comment:
I would consider following:

```
prototype(Neos.NeosIo:VideoEmbed) < prototype(Neos.Neos:ContentComponent) {
    videoFileTypes = ${['webm', 'mp4', 'ogg']}
    assets = ${q(node).property('assets')}

    renderer = Neos.Fusion:Component {
        sources = Neos.Fusion:Loop {
            items = ${props.videoFileTypes}
            itemName = 'fileextension'
            itemRenderer = Neos.Fusion:Loop {
                items = ${props.assets}
                itemName = 'asset'
                itemRenderer = afx`
                    <source @if.isRequiredFiletype={asset.resource.fileextension == fileextension} type={'video/' + fileextension}>
                        <Neos.Fusion:ResourceUri @path='attributes.src' resource={asset.resource} />
                    </source>
                `
            }
        }
        renderer = afx`
             <video @if.set={props.sources} class="videoEmbed" autoplay muted loop="true" playsinline="true">
                {props.sources}
            </video>
            <p @if.set={!props.sources}>Please select a video in the inspector</p>
        `
    }
}
```

* The user can add multiple files
* The get sorted in the right way, regardless how the user put them into the inspector
* All other filetypes than mp4, ogg or webm get ignored.